### DISCUSSION TITLE
提示出现错误

### DISCUSSION THREAD

**Discussion by pinox233** [ywmoyue/biliuwp-lite#649] [Q&A]
### Bug 描述

请查看日志

### 操作步骤

```plain text
打开出现且uwp无法使用
```

### 期望结果

修复

### 日志

[20240523.log](https://github.com/ywmoyue/biliuwp-lite/files/15416506/20240523.log)

### 应用来源

Github/其他Github镜像站

### 备注

1

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
看起来是所有的网络连接都报错了。 
请先试一下重新安装本应用，
还有一种可能就是uwp loopback, 查看 https://zhuanlan.zhihu.com/p/363912808 试试其中的解决方法。

**Comment by ywmoyue** [comment]
请检查设备上其他uwp应用是否有网

**Comment by ywmoyue** [comment]
@pinox233 是否解决了问题呢
